node/cn/backend.go

```diff
@@ -221,6 +221,17 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	config.GasPrice = new(big.Int).SetUint64(chainConfig.UnitPrice)
 	logger.Info("Initialised chain configuration", "config", chainConfig)
 
+	// TODO-klaytn set KIP71 forked block number
+	// It's intended not to modify something like genesis in SetupGenesisBlock()
+	chainConfig.KIP71CompatibleBlock = params.DefaultKIP71CompatibleBlockNum
```

Comment:
Do you mean to always overwrite genesis's KIP71CompatibleBlock value?
It might overwrite the value from homi-generated genesis.json if any.

I'm asking because `CompatibleBlock = ` pattern does not appear in anywhere in our codebase except `cmd/homi/setup/cmd.go`.

Reply:
You are right. It's our first hard fork by hard coded block number as I know.

It can be resolved by following
```go
if chainConfig.KIP71CompatibleBlock == nil {
  chainConfig.KIP71CompatibleBlock = params.DefaultKIP71CompatibleBlockNum
}
```
I reviewed the code about `cmd/homi/setup/cmd.go` for KIP71 config but didn't fine any use case.
I think the homi support for KIP71 is another issue. Do you think it's better to remove the code about homi in this pr?

Reply:
- If it's our first hard fork by hardcoded block number, then

  ```
  if chainConfig.KIP71CompatibleBlock == nil {
    chainConfig.KIP71CompatibleBlock = params.DefaultKIP71CompatibleBlockNum
  }
  ```
  seems good to me.

- No I didn't mean to say we should modify homi. Sorry for the confusion.

Reply:
In my view, hardcoding the forked block number seems not bad. Any block number can be interpreted as a KIP17-compatable block number If the number is grabbed from `genesis.json`. I'm not sure the source code is a reasonable place to record the history. I would appreciate letting me know if there's a traditional process.